packages/apps/kadena-docs/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@kadena/kadena-docs",
+  "version": "0.1.0",
+  "scripts": {
+    "_phase:build": "next build",
+    "_phase:test": "",
+    "build": "next build",
+    "cypress:open": "cypress open",
+    "dev": "next dev",
+    "lint": "next lint",
+    "start": "next start",
+    "storybook": "storybook dev -p 6006",
+    "test": "jest"
+  },
+  "dependencies": {
```

Comment:
Can you add the lint stuff, see: https://github.com/kadena-community/kadena.js/blob/master/packages/apps/explorer/package.json#L11-L24